src/middlewared/middlewared/plugins/device_/device_info_linux.py

```diff
@@ -174,27 +202,21 @@ def get_disk_details(self, block_device, disk, lshw_disks):
                 if reg:
                     disk['serial'] = disk['ident'] = reg.group(1)
 
-        if not disk['model'] and os.path.exists(os.path.join(disk_sys_path, 'device/model')):
-            # For nvme drives, we are unable to retrieve it via lshw
-            with open(os.path.join(disk_sys_path, 'device/model'), 'r') as f:
-                disk['model'] = disk['descr'] = f.read().strip()
-
-        # We make a device ID query to get DEVICE ID VPD page of the drive if available and then use that identifier
-        # as the lunid - FreeBSD does the same, however it defaults to other schemes if this is unavailable
-        lun_id_cp = subprocess.Popen(
-            ['sg_vpd', '--quiet', '-i', device_path],
-            stdout=subprocess.PIPE, stderr=subprocess.PIPE,
-        )
-        cp_stdout, cp_stderr = lun_id_cp.communicate()
-        if not lun_id_cp.returncode and lun_id_cp.stdout:
-            lunid = cp_stdout.decode().strip()
-            if lunid:
-                disk['lunid'] = lunid.split()[0]
-            if lunid and disk['lunid'].startswith('0x'):
-                disk['lunid'] = disk['lunid'][2:]
-
-        if disk['serial'] and disk['lunid']:
```

Comment:
Can we please just have this and remove the if block at L183 in the new diff ? As if we get into last `if` block, we will miss setting `serial_lunid`.